tests/test_qr_to_waypoint_names.py

```diff
@@ -0,0 +1,60 @@
+"""
+Testcases for parsing QR to waypoint.
+"""
+
+
+from modules.qr_to_waypoint_names import qr_to_waypoint_names
+
+
+def test_valid_waypoint_single():
+    valid, data = qr_to_waypoint_names("Follow route: Alpha")
```

Comment:
Convention for tests is to have 3 sections: Setup, run, test, with expected and actual for expected output and actual output respectively.

```py
# Setup
input = "Follow route: Alpha"
expected == ["Alpha"]

# Run
result, actual = qr_to_waypoint_names(input)

# Test
assert result
assert actual == expected
```

Tests should be single units (unit test). Each test would have its own name (e.g. this would be split into possibly test_valid_single_simple and test_valid_single_with_surrounding_whitespace ). Don't be afraid of long test case names.

Reply:
I will write documentation on unit tests someday.